WcaOnRails/config/i18n-tasks.yml.erb

```diff
@@ -0,0 +1,135 @@
+# i18n-tasks finds and manages missing and unused translations: https://github.com/glebm/i18n-tasks
+
+# The "main" locale.
+base_locale: en
+## All available locales are inferred from the data by default. Alternatively, specify them explicitly:
+# locales: [es, fr]
+## Reporting locale, default: en. Available: en, ru.
+# internal_locale: en
+
+# Read and write translations.
+data:
+  ## Translations are read from the file system. Supported format: YAML, JSON.
+  ## Provide a custom adapter:
+  # adapter: I18n::Tasks::Data::FileSystem
+
+  # Locale files or `File.find` patterns where translations are read from:
+  read:
+    ## Default:
+    - config/locales/%{locale}.yml
+    ## More files:
+    - config/locales/devise.views.%{locale}.yml
+    - config/locales/doorkeeper.%{locale}.yml
+    - config/locales/kaminari.%{locale}.yml
+    - config/locales/rails.%{locale}.yml
+    - config/locales/time_will_tell.%{locale}.yml
+    - config/locales/carrierwave.%{locale}.yml
```

Comment:
The idea behind this was to specify exactly the locale file we intend to provide, but you're right it's just simpler that way.